src/compiler/parser-decl-symbtype.bas

```diff
@@ -393,6 +393,38 @@ function hIntegerTypeFromBitSize _
 
 end function
 
+private function cMangleModifier _
+	( _
+	) as integer
+
+	static as zstring * FB_MAXNAMELEN+1 aliasid
+	function = FALSE
+
+	'' ALIAS?
+	if( lexGetToken( ) = FB_TK_ALIAS ) then
+		lexSkipToken( )
+
+		'' "long"
+		if( lexGetClass( ) = FB_TKCLASS_STRLITERAL ) then
+			aliasid = *lexGetText( )
```

Comment:
The aliasid static var seems unnecessary, since it's only used locally - I think it would be good to eliminate it (by checking the string immediately after lexGetText() and moving the lexSkipToken() behind the check).

Reply:
Removed the static aliasid,  Yea, an earlier version was returning a string, but that turned out to be not very useful.  Thanks.